pyaedt/edb_core/edb_data/padstacks_data.py

```diff
@@ -1062,26 +1062,31 @@ def backdrill_top(self):
         Returns
         -------
         tuple
-            Tuple of the layer name and drill diameter.
+            Tuple of the layer name, drill diameter and offset if it exists.
```

Comment:
```suggestion
            Tuple of the layer name, drill diameter, and offset if it exists.
```
We always use the Oxford comma, which simply means placing a comma before the "and" or "or" in a list of items.